templates/zerver/for-companies.html

```diff
@@ -334,8 +334,7 @@ <h1>
                             call</a> with the click of a button, or
                             make plans for later without worrying
                             about time zones
-                            using <a href="/help/format-your-message-using-markdown#mention-a-time">local
-                            times</a>.
+                            using <a href="/help/format-your-message-using-markdown#global-time">global time</a>.
```

Comment:
This feels like weird English; do we want "using a global time" or "using global times"?  Probably the latter, which is what we had before.

Reply:
I think this comment applies to every instances; checked quickly with @alya and she agrees; can you tweak?